etc/ipsec.conf

```diff
@@ -16,9 +16,9 @@ conn %default
 conn rw
     # http://wiki.loopop.net/doku.php?id=server:vpn:strongswanonopenvz
     # https://wiki.strongswan.org/projects/strongswan/wiki/ForwardingAndSplitTunneling
-    leftsubnet=0.0.0.0/0
+    leftsubnet=0.0.0.0/0,::/0
     # end ref
     leftfirewall=yes
     right=%any
-    rightsourceip=10.8.0.0/16
+    rightsourceip=10.8.0.0/16,2a00:1450:400c:c05::/64
```

Comment:
The IP range 2a00:1450:400c:c05::/64 seems not in the range of reserved IP addresses, please update it to use private IP range, I thinks the IPv4 mapped addresses section is quite applicable.